### PR TITLE
Improvement/ember cli mirage

### PR DESCRIPTION
**Pull request by ultimatemonty**
@gschorkopf @pdougall1 this sets up some super basic mirage routes to enable devs to at least get the site loading locally without the Rails API running. Lots still to be done but this is a start

### CODE REVIEW

File: app/templates/application.hbs [hunk 1]
```diff
@@ -50,6 +50,13 @@
           </a>
         {{/if}}
         {{link-to "Frolfr" "index" id="title" classNames="navbar-brand navbar-brand-text navbar-left navbar-link"}}
+        {{#if currentUser.isAuthenticated}}
```
Comment (gschorkopf) [inline app/templates/application.hbs]: Why was this commit included in the PR?

File: ember-cli-build.js [hunk 2]
```diff
@@ -4,21 +4,27 @@ var EmberApp = require('ember-cli/lib/broccoli/ember-app');
...
+  app.import('bower_components/ember-uploader/dist/ember-uploader.js');
+  app.import('bower_components/floatThead/dist/jquery.floatThead.js');
+
+
+  var fontAwesome = pickFiles('bower_components/components-font-awesome/fonts', {
+      srcDir: '/',
+      destDir: '/fonts'
+  });
+
+  return app.toTree([fontAwesome]);
```
Comment (gschorkopf) [inline ember-cli-build.js:39]: This commit's changes are also made here: https://github.com/frolfr/frolfr-client/pull/41/commits/ddbc2ec70053cc552cf34b3ab58cd6724ff2cb93

We'll have to rebase this eventually!

**Comment by gschorkopf** [comment]
Looks good!

**Comment by gschorkopf** [comment]
I figured we'd run into some merge conflicts after #41

**Comment by ultimatemonty** [comment]
@gschorkopf removed the template commit and the ember-cli-build commit, and merge the upstream master. Should be good to go now

**Comment by gschorkopf** [comment]
Seems legit! Thanks for getting this started!

**Pull request merged**